l10n_ch_lsv_dd/model/move_line.py

```diff
@@ -39,3 +39,33 @@ def _is_generate_bvr(self, cr, uid, invoice, context=None):
         return val or (invoice.partner_bank_id and
                        invoice.partner_bank_id.state == 'iban' and
                        invoice.partner_bank_id.lsv_identifier)
+
+    def line2bank(self, cr, uid, ids, payment_mode_id, context=None):

```

Comment:
Where is initially declared this function ? I wonder if it would not make sense to make the correction directly in the super method ?